misc/acrn-manager/README.rst

```diff
@@ -107,6 +107,12 @@ Use the ``stop`` command to stop one or more running VM:
 
    # acrnctl stop vm-yocto vm1-14:59:30 vm-android
 
+Use the optional ``-f`` or ``--force`` argument to force the stop operation.
```

Comment:
Just added more explanation and force-pushed the updated commit. Let me know if this addresses your comment.

@taoyuhong , please review and jump in in case my explanation in incorrect!

Reply:
Yes, this is my understanding based on the following commit messages:
* https://github.com/projectacrn/acrn-hypervisor/commit/59fd4202d3e392044b42e90fda2f0999e6376dcd
* https://github.com/projectacrn/acrn-hypervisor/commit/653aa859b1de0195f7c6312fff6f3596d5305d73
* https://github.com/projectacrn/acrn-hypervisor/commit/8b27daa7789773d201652c406cf1084f9834a1b8

Thanks